docs/getting-started.html

```diff
@@ -0,0 +1,40 @@
+<html>
+  <head>
+    <title>
+      Manubot - Docs - Getting Started
+    </title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Quicksand"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Inconsolata"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="../styles.css" />
+  </head>
+  <body>
+    <header style="height: 100%">
+      <video width="100%" controls autoplay muted>
+        <source src="getting-started.mp4" type="video/mp4" >
```

Comment:
This video is 94 MB... a bit large for tracking with git, close to GitHub's limit of 100 MB. Can we just use the mp4 from Figshare so we don't have to track it in this repo?

Git LFS would also be another alternative (never used it with GitHub Pages). We could use LFS for all binary files. Update: this does not work well as per https://github.com/git-lfs/git-lfs/issues/1342